test/system/005-info.bats

```diff
@@ -187,6 +187,25 @@ host.slirp4netns.executable | $expr_path
     fi
 }
 
+@test "rootless podman with symlinked $HOME" {
+    if ! is_remote; then
```

Comment:
this should call skip "..." to not claim that this test is doing anything in the log.